### PR TITLE
remove internal run

### PR DESCRIPTION
**Pull request by macklin-10x**
Removes the internal_run flag and everything related to the config system.

### CODE REVIEW

File: enclone_main/src/determine_ref.rs [hunk 1]
```diff
@@ -83,43 +83,6 @@ pub fn determine_ref(ctl: &mut EncloneControl, refx: &mut String) -> Result<(),
...
-            if ctl.gen_opt.descrip {
-                println!("using IMGT human reference");
-            }
-            let f = open_for_read![imgt];
-            for line in f.lines() {
-                let mut s = line.unwrap();
-                if ctl.gen_opt.imgt_fix {
-                    // Fix IGHJ6.
-                    if s == *"ATTACTACTACTACTACGGTATGGACGTCTGGGGCCAAGGGACCACGGTCACCGTCTCCTCA"
-                        || s == *"ATTACTACTACTACTACTACATGGACGTCTGGGGCAAAGGGACCACGGTCACCGTCTCCTCA"
```
Comment (sreenathkrishnan) [inline enclone_main/src/determine_ref.rs:100]: Hmm, this looks like potentially useful information about the reference buried here

File: enclone_paper/src/bin/build_clonotyping_inputs.rs [hunk 2]
```diff
@@ -46,17 +45,18 @@ pub fn main() {
...
+    // let mut config_file = String::new();
+    // for (key, value) in env::vars() {
+    //     if key == "ENCLONE_CONFIG" {
+    //         config_file = value.to_string();
+    //         if config_file.contains(',') {
+    //             config_file = config_file.after(",").to_string();
+    //         }
+    //     }
+    // }
+    // let _ = get_config(&config_file, &mut config);
```
Comment (sreenathkrishnan) [inline enclone_paper/src/bin/build_clonotyping_inputs.rs:59]: No point keeping these lines?

File: enclone_tools/src/bin/release_nanny.rs [hunk 3]
```diff
@@ -79,16 +78,17 @@ fn main() {
...
+        // for (key, value) in env::vars() {
+        //     if key == "ENCLONE_CONFIG" {
+        //         config_file = value.to_string();
+        //     }
+        // }
         let mut remote_version_file = String::new();
         let mut remote_version = None;
-        if get_config(&config_file, &mut config) {
+        // if get_config(&config_file, &mut config) {
+        if false {
```
Comment (sreenathkrishnan) [inline enclone_tools/src/bin/release_nanny.rs:91]: Should we remove this block too?

**Review by sreenathkrishnan — APPROVED** [review]

File: enclone_main/src/determine_ref.rs (hunk 1 above)
Comment (macklin-10x) [reply enclone_main/src/determine_ref.rs:100]: Can we archive this somewhere useful?

File: enclone_paper/src/bin/build_clonotyping_inputs.rs (hunk 2 above)
Comment (macklin-10x) [reply enclone_paper/src/bin/build_clonotyping_inputs.rs:59]: I opted to leave them in place; this entire executable will never run again, so we should either attempt to preserve it even in the face of refactoring, or go ahead and delete the entire thing.

File: enclone_tools/src/bin/release_nanny.rs (hunk 3 above)
Comment (macklin-10x) [reply enclone_tools/src/bin/release_nanny.rs:91]: Same thing here - we're probably never going to run this code again.  If I start deleting code from here, it results in a cascade of "unused" vars.

**Pull request merged**